mne/annotations.py

```diff
@@ -56,6 +47,15 @@ class Annotations(object):
         In general, ``raw.info['meas_date']`` (or None) can be used for syncing
         the annotations with raw data if their acquisiton is started at the
         same time.
+    onset : array of float, shape (n_annotations,)
+        The starting time of annotation in seconds after ``orig_time``.
```

Comment:
annotation is missing an s.  
```diff
- annotation
+ annotations
```

Reply:
here's a nice comment about this comment